tests/integration.rs

```diff
@@ -5854,6 +6001,86 @@ mod tests {
 
             handle_child_output(r, &output);
         }
+
+        #[test]
+        #[cfg(not(feature = "mshv"))]
+        fn test_windows_guest_disk_hotplug() {
+            let windows_guest = WindowsGuest::new();
+
+            let mut ovmf_path = dirs::home_dir().unwrap();
+            ovmf_path.push("workloads");
+            ovmf_path.push(OVMF_NAME);
+
+            let tmp_dir = TempDir::new_with_prefix("/tmp/ch").unwrap();
+            let api_socket = temp_api_path(&tmp_dir);
+
+            let mut child = GuestCommand::new(windows_guest.guest())
+                .args(&["--api-socket", &api_socket])
+                .args(&["--cpus", "boot=2,kvm_hyperv=on"])
```

Comment:
Although it does no harm, `kvm_hyperv=on` is not required for MSHV. You don't need to change your patch just because of this. I've taken a note to go through the test cases to remove it to avoid confusion.

Reply:
ACK, thanks for pointing to tihs. Making it more centralized would sure make sense, like a function that would deliver this part depending on mshv feature so then it could be reused later on.

Thanks

Reply:
@liuw but the code is correct here since the tests are not being run for MSHV, which means they're run with KVM only, which means they need `kvm_hyperv=on`.

Reply:
I was thinking about having `kvm_hyperv=on` guarded by `kvm`.

Reply:
Sounds good :+1: